mpisppy/cylinders/ph_ob.py

```diff
@@ -89,6 +90,7 @@ def _phboundsolve(self, iternum):
             tee=teeme,
             verbose=verbose,
             warmstart=warmstart,
+            need_solution=False,
         )
```

Comment:
If there is no solution, what happens to the bounds that solve_one grabs, which it does whether there was a solution or not.  Meanwhile, And Ebound just computes using whatever bounds solve_one has put in there. There is probably something I am missing, but it makes me nervous.